### ISSUE TITLE
escapistmagazine.com: breakage

### ISSUE THREAD

**Issue by GlacierIsland** [uBlockOrigin/uAssets#15424]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.escapistmagazine.com/a-plague-tale-requiem-zero-punctuation/`

### Category

breakage

### Describe the issue

The video player is not visible with any adblocker enabled.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Screenshot 2022-10-26 134224](https://user-images.githubusercontent.com/5401245/198109757-deacb46b-7c3f-4475-8d9b-1deff2186929.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.44.4
Chromium: 106
filterset (summary): 
  network: 247369
  cosmetic: 130738
  scriptlet: 36805
  html: 0
listset (total-discarded, last updated): 
  removed: 
    plowe-0: null
  added: 
    adguard-annoyance: 60059-91, 2d.16h.16m
    adguard-generic: 64205-473, 2d.16h.14m
    adguard-mobile: 8174-45, 2d.16h.18m
    adguard-spyware: 33737-2530, 2d.16h.15m
    adguard-spyware-url: 793-1, 2d.16h.17m
    block-lan: 44-0, 20d.23h.12m
    curben-phishing: 120766-0, 2h.36m
    curben-pup: 192-1, 2h.37m
    ublock-annoyances: 4856-190, 2d.16h.19m
  default: 
    user-filters: 0-0, never
    easylist: 62003-967, 1d.23h.24m
    easyprivacy: 30777-12798, 1d.23h.23m
    ublock-abuse: 76-0, 2h.38m
    ublock-badware: 4357-242, 2d.16h.13m
    ublock-filters: 33190-3832, 2d.16h.12m
    ublock-privacy: 260-42, 1d.9h.46m
    ublock-quick-fixes: 418-33, 2h.35m
    ublock-unbreak: 1877-96, 2d.16h.11m
    urlhaus-1: 10608-7, 2h.34m
filterset (user): [empty]
trustedset: 
  added: [array of 2 redacted]
switchRuleset: 
  added: [array of 1 redacted]
modifiedUserSettings: 
  cloudStorageEnabled: true
  colorBlindFriendly: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 408 ms (selfie)
  maxAssetCacheWait: 281 ms
popupPanel: 
  blocked: 12
  blockedDetails: 
    escapistmagazine.com: 1
    brid.tv: 2
    chartbeat.com: 2
    google-analytics.com: 1
    googletagmanager.com: 3
    pubnation.com: 1
    unpkg.com: 1
    wp.com: 1
```
</details>

**Comment by okiehsch** [comment]
Works fine with a default setup.

**Comment by hellscyth142** [comment]
Found this entry in the issue tracker off google. Unfortunatley, google being shit these days, I had to specifically use reddit search to find this thread: https://www.reddit.com/r/uBlockOrigin/comments/xwf540/anyone_else_having_problems_with_escapistmagazine/

From someone in the thread's testing, they found Web Annoyances Ultralist was blocking the videos. I didn't have that one isntalled, so I did my own testing. The culprit for me was Fanboy’s Annoyance.

So for anyone else who stumbles into this thread hoping for an answer, there you go. Turn those off and it should work.
